learn/arvo/hoon/hoon-syntax.udon

```diff
@@ -36,18 +36,18 @@ Hoon source files are composed almost entirely of the [printable ASCII character
 Hoon code makes heavy use of non-alphanumeric symbols.  Reading off code aloud using the proper names of ASCII symbols is tedious, so we've mapped syllables to symbols:
 
 ```
-ace  [1 space]       hep  -            rob  }
-ban  >               ket  ^            say  '
-bar  |               lac  [            sig  ~
-bat  \               led  <            tar  *
-bus  $               lit  (            tec  `
-cab  _               lob  {            tis  =
-cen  %               lus  +            vat  @
-col  :               mic  ;            wut  ?
```

Comment:
though I will note stylistically that "mic" is alphabetically after "lus", not "sel"